### PR TITLE
COMMERCE-6436 commerce-product-service changed FriendlyURLNormalizerUtil method call

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://issues.liferay.com/browse/COMMERCE-6436

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Review by dsitu — APPROVED** [review]

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPOptionLocalServiceImpl.java [hunk 1]
```diff
@@ -81,7 +81,7 @@ public CPOption addCPOption(
 			externalReferenceCode = null;
 		}
 
-		key = FriendlyURLNormalizerUtil.normalize(key);
+		key = FriendlyURLNormalizerUtil.normalizeWithPeriodsAndSlashes(key);
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPOptionLocalServiceImpl.java:84]: add integration tests

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Pull request closed without merging**